ci-operator/step-registry/gather/must-gather/gather-must-gather-commands.sh

```diff
@@ -4,6 +4,41 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+function createIntallJunit() {
+  if test -f "${SHARED_DIR}/install-status.txt"
+  then
+    EXIT_CODE=`cat ${SHARED_DIR}/install-status.txt | awk '{print $1}'`
+    INSTALL_STATUS=`cat ${SHARED_DIR}/install-status.txt | awk '{print $2}'`
+    if [ "$EXIT_CODE" ==  0  ];then
+      cat >"${ARTIFACT_DIR}/junit_install.xml" <<EOF
+      <testsuite name="install" tests="2" failures="0">
+        <testcase name="install test"/>
+        <testcase name="bootstrap test"/>
+      </testsuite>
+EOF
+    elif [ $INSTALL_STATUS == "bootstrap" ] || [ $INSTALL_STATUS == "cluster_created" ]
+    then
+      cat >"${ARTIFACT_DIR}/junit_install.xml" <<EOF
+      <testsuite name="install" tests="2" failures="1">
+        <testcase name="install test">
+          <failure message="">openshift install failed after stage $INSTALL_STATUS</failure>
+        </testcase>
+        <testcase name="bootstrap test"/>
+EOF
+    else
```

Comment:
if install_status says "manifests", why does that mean the openshift-install failed as opposed openshift-install was never attemped?

Reply:
I consider the whole install script here as the test step. If manifests portion fails, the overall status will fail. Therefore, I use the junit to represent that failure.